src/LazyLoadImage/LazyLoadImage.types.ts

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+/**
+ * Props for the LazyLoadImg component
+ */
+export type LazyLoadImageProps = {
+  /**
+   * Source of the image
+   */
+  src: string;
+  /**
+   * Alternative text for the image
+   */
+  alt: string;
```

Comment:
Is `alt` really need to be mandatory @dmbarry86 ? Is so we dont need this check inside  component `alt={alt || ""}`

Reply:
Yes, alt is mandatory for `<img>`, otherwise you will get a warning. Whether that is implemented through mandatory type or optional type with some default, I don't mind.